docs/contributors/release.md

```diff
@@ -56,17 +56,17 @@ Releasing the first release candidate for this milestone (`x.x`) involves:
 To generate a changelog for a release, use the changelog generator tool:
 
 ```
-node bin/changelog.js
+node bin/plugin/cli.js changelog
```

Comment:
I can't recall there was any specific reason to not just promote the NPM script directly:

```suggestion
npm run changelog
```

Reply:
I think we should have an npm script for the plugin CLI (release tool) and not directly for a given command and if you want help you'd do: `npx cli --help` or something like that.

Reply:
I didn't find a nice way to apply my proposal above yet (we might need to first move this to a package). I'll explore later, I used your suggestion now.